Avoid using the _intel suffix name for library
We can only have 1 library per version regardless, and the ABI is
determined by the toolchain like with all other stuff we compile.

Author: micketeer@gmail.com
--- CMakeLists.txt.orig	2022-11-28 18:35:05.033932657 +0000
+++ CMakeLists.txt	2022-11-28 18:35:20.226889430 +0000
@@ -167,7 +167,7 @@
 ENDIF()
 
 IF ( ABI STREQUAL "Intel")
-	SET(FLEXIBLAS_ABI_INTEL TRUE)
+	SET(FLEXIBLAS_ABI_INTEL FALSE)
 	SET(USE_INTERFACE_INTEL TRUE)
 ELSEIF ( ABI STREQUAL "XL" )
 	SET(FLEXIBLAS_ABI_IBM TRUE)
